.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- Don't forget a short, descriptive title! -->
+
+# What's in this PR?
+
+<!-- List the changes you made and your reasons for them.-->
+
+<!-- Make sure any changes to code include changes to documentatino.-->
```

Comment:
That's just based off markdown not rendering comments.so in the edit tab it'll have all the markdown including the comments from this file, then once they hit the preview tab or as it'll appear when published, markdown won't compile the comments.